packages/db/fauna/schema.graphql

```diff
@@ -195,43 +196,74 @@ type Upload {
 }
 
 """
-Filecoin deal for a pin.
+A batch of content for inclusion in a Filecoin deal.
 """
-type Deal {
+type Batch {
   """
-  CIDs in this deal.
+  CID of the root of the batch data.
   """
-  contents: [Content!]! @relation
+  cid: String! @unique
 
   """
-  ID of miner this deal was made with.
+  Piece CID.
   """
-  miner: String
+  pieceCid: String
```

Comment:
CreateBatchInput has a non-nullable pieceCID, so this could be non-nullable too, if we care.
```suggestion
  pieceCid: String!
```